cobalt/browser/h5vcc_metrics/public/mojom/h5vcc_metrics.mojom

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module h5vcc_metrics.mojom;
+
+interface MetricsListener {
+  OnMetrics(string tbd);
+};
+
+interface H5vccMetrics {
+  AddListener(pending_remote<MetricsListener> listener);
+
+  Enable() => ();
```

Comment:
These aren't implemented yet, correct? Does it make sense/is it possible for us to store this value in the H5vccMetrics class (in the renderer) and just send a request to enable/disable to the platform? That way we could implement isEnabled purely in H5vccMetrics

Reply:
If we trust `Enable()`s result to be sticky in the browser side then we could keep it purely as a mirror in Blink side, yes. Would you like that to happen now or in the next CL?

Reply:
I think for now let's mark isEnabled as sync in the idl and remove IsEnabled from this mojom file, and if we decide we need it async when we actually implement we can make that change. Does that sound okay?

Reply:
Done, I think: last commit changes the Mojom interface by collapsing Enable/Disable into a single Sync call, and tracks enabled/disabled state via flag in IDL impl code; it also adds a add readonly attribute |enabled| to the IDL def.